Add flag to show a prompt when closing mutitab (#1670)
Eloston/ungoogled-chromium: docs/flags.md

  <code>Switch&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</code> | Description
  -- | --
  `--bookmark-bar-ntp` | Sets the visibility of the bookmark bar on the New Tab Page. Only takes the value `never`.

>   `--close-confirmation` | Show a warning prompt when closing the browser window. Accepts `last` (when closing the last window with multiple tabs) and `all` (when more than one windows are open). 

Reviewer: ```suggestion
  `--close-confirmation` | Show a warning prompt when closing the browser window. Accepts `last` (prompt when closing the last window with several tabs) and `multiple` (prompt only if more than one window is open). 
```
